webextensions/api/webRequest.json

```diff
@@ -19,7 +19,8 @@
             },
             "opera": "mirror",
             "safari": {
-              "version_added": "14"
+              "version_added": "14",
+              "notes": "Support for use in Manifest V3 extensions, that is with non-persistent background pages/scripts, provided in Safari 18."
             },
             "safari_ios": "mirror"
```

Comment:
iOS should be "18" since it wasn't supported earlier, as iOS requires non-persistent background content.